lib/tojson.js

```diff
@@ -194,6 +204,10 @@ exports.toMediaJSON = function (media, session, opts) {
         trans.fingerprints.push(fp);
     });
 
+    if (parsers.findLine('a=ice-options:trickle', lines)) {
```

Comment:
i'd do this in a separate PR/commit for atomicity

Reply:
actually this shouldn't be mapped. First its not really relevant in the SDP, second, as xep-0371 explains we have disco for this kind of stuff.